app/utils/validators.js

```diff
@@ -70,6 +70,11 @@ export const validGithubProfileUrlPattern = new RegExp(
   + '?(?:www.)?github\\.com\\/([a-zA-Z0-9_]+)([-]?)([a-zA-Z0-9_]+)$'
 );
 
+export const validInstagramProfileUrlPattern = new RegExp(
+  '^(https?:\\/\\/)' // compulsory protocol
+  + '?(?:www.)?(instagram\\.com\\/)([a-zA-Z0-9_.]+)$'
```

Comment:
```suggestion
  + '?(www.)?(?:instagram.com|instagr.am)\/[a-zA-Z0-9_.]+'
```

Reply:
Thanks for the suggestion. I assumed that.